src/features/src/changeVolumeFeature.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import Slider from '../../clui-ui-components/Slider'
+import List from '../../clui-ui-components/List'
+
+import FFmpegFeature from '../FFmpegFeature'
+
+type ChangeVolumeConfig = {
+  VOLUME: { value: number; [name: string]: any }
+  [name: string]: any
+}
+
+class ChangeVolumeFeature extends FFmpegFeature {
+  configuration: ChangeVolumeConfig
+
+  constructor(configuration: ChangeVolumeConfig) {
+    super()
+    this.configuration = configuration
+    const { volumeValue } = this.parseConfiguration()
+    const command = this.changeVolumeCommand(volumeValue)
+    this.setFFmpegCommands(command)
+    this.setProgress()
+    this.setFileConfig()
+  }
+
+  parseConfiguration = () => {
+    const { VOLUME } = this.configuration
+    const { VALUE } = this.configuration
+    console.warn('IMPORTANT!!!! ', VOLUME, 'rgt', VALUE)
+    return { volumeValue: VALUE }
+  }
+
+  setFFmpegCommands(command: string) {
+    this.ffmpegCommands = command
+  }
+
+  setProgress = () => {
+    this.progressBar.name = 'Changing Volume...'
+    this.progressBar.color = '#3FBD71'
+  }
+
+  setFileConfig = () => {
+    this.fileConfig = {
+      types: [{ name: 'video', number: { min: 1, max: 1 } }],
+      primaryType: 'video'
+    }
+  }
+
+  /**
+   * Sets the volume value in the FFmpeg command
+   * @param volumeValue: The volume value chosen on the slider
+   */
+  private changeVolumeCommand = (volumeValue: number): string => {
+    const decimalVolume = volumeValue / 100
+    const finalVolumeValue = `-af 'volume=${decimalVolume}'`
```

Comment:
Should mention where this command comes from and the documentation of it, to verify if it is being used correctly

Reply:
Will include, though it wasn't necessary because of other setFFmpegCommands() implementations

Reply:
Fixed it